### PR TITLE
{debugger}[foss/2016b] heaptrack v1.1.0

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

Requires ~~#6449~~ + ~~#6452~~

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
cermis - Linux debian 9.4, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/377379106393e074f8e921f8e0629503 for a full test report.

File: easybuild/easyconfigs/h/heaptrack/heaptrack-1.1.0-foss-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+dependencies = [
+    ('zlib', '1.2.8'),
+    ('zstd', '1.3.4'),
+    ('Boost', '1.63.0', '-Python-2.7.12'),
+    ('libunwind', '1.2.1'),
+    ('libdwarf', '20150310'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/heaptrack', 'bin/heaptrack_print', 'lib/heaptrack/libheaptrack_preload.so'],
```
Comment (boegel) [inline easybuild/easyconfigs/h/heaptrack/heaptrack-1.1.0-foss-2016b.eb]: @vanzod Please use `SHLIB_EXT` rather than hardcoding `.so`

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 4 (4 easyconfigs in this PR)
node2024.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/0a3c89c1298509667a458ae7097d989c for a full test report.

File: easybuild/easyconfigs/h/heaptrack/heaptrack-1.1.0-foss-2016b.eb (hunk 1 above)
Comment (vanzod) [reply easybuild/easyconfigs/h/heaptrack/heaptrack-1.1.0-foss-2016b.eb]: Good catch. I forgot that...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2717.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/36c707dac9b22a2a066dc16f6f5db8be for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
gligar03.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f32edb4e9d5f9694a30601e8d44189f4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2025.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4357f8aa45b870a6257a3c8e6c82424e for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**